eisbuk-admin/src/components/atoms/DateSwitcher/DateSwitcher.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { DateTime, DateTimeUnit } from "luxon";
+import { useDispatch } from "react-redux";
+
+import { CalendarPicker } from "@mui/lab";
+
+import { changeCalendarDate } from "@/store/actions/appActions";
+import Menu from "@mui/material/Menu";
+
+interface Props {
+  currentDate: DateTime;
+  /**
+   * Value for next/prev timeframe start time calculation
+   */
+  jump?: DateTimeUnit;
+  anchorEl: null | HTMLElement;
+  open: boolean;
+  handleClose: () => void;
```

Comment:
Here you can simply extend the aforementioned `MenuProps`:
```typescript
interface Props extends MenuProps {
  jump: DateTimeUnit // good one btw ;)
  currentDate: DateTime
}
```

Reply:
This way the `onClose`, `open` and `anchorEl` will be included in the props interface. About `onClose` / `handleClose`, it is a convention to use `handle<something>` (i.e. `handleClick`) when you're writing a handler within the component, while all handlers received as props should be `on<something>` (i.e. `onClick`)

Example:
```typescript
const Button = ({ onClick, children }) => <button {...{ onClick }} >{children}</button>

const ButtonBox = () => {
  const handleClick = () => {/* do something */}
  return <Button onClick={handleClick} >Click Me</Button> 
}
```